e2e/wdio/platform/tests/input.e2e-spec.ts

```diff
@@ -40,13 +40,13 @@ describe('Input should ', function() {
     });
 
     it('have input without label', () => {
-        waitElementToBePresentInDOM(inputPage.defaultInput);
+        waitForPresent(inputPage.defaultInput);
         waitForElDisplayed(inputPage.defaultInput);
         expect(doesItExist(inputPage.autocompleteInputLabel)).toBe(false);
     });
 
     it('be able to type something with keyboard', () => {
-        waitElementToBePresentInDOM(inputPage.defaultInput);
+        waitForPresent(inputPage.defaultInput);
         waitForElDisplayed(inputPage.defaultInput);
```

Comment:
if `waitForElDisplayed` used mostly with `waitForPresent`, then maybe `waitForPresent` should include `waitForPresent` inside of it? to reduce boiler plate?

Reply:
fixed